index.html

```diff
@@ -1,278 +1,375 @@
+                <span class="project-name">project name goes here</span>
+                <div>
+                  <ul class="project-languages">
+                    <li class="language-css">HTML/CSS</li>
+                    <li class="language-css">Ruby on Rails</li>
+                    <li class="language-css">JavaScript</li>
+                  </ul>
+                </div>
+                <button type="button" class="project-button">
+                  <span>See this project</span>
+                  <img
+                    src="assets/images/arrow.svg"
+                    alt="arrow"
+                    class="arrow"
+                  />
+                </button>
+              </div>
+            </div>
+          </section>
 
-                    <section class="containing-6">
-                        <div class="rectangle_21">
-                            <img src="assets/images/rectangle_24.svg" alt="rectangle_21" />
-                        </div>
-                        <div class="rectangle_22">
-                            <span class="project-name">project name goes here</span>
-                            <div>
-                                <ul class="project-languages">
-                                    <li class="language-css">HTML/CSS</li>
-                                    <li class="language-css">Ruby on Rails</li>
-                                    <li class="language-css">JavaScript</li>
-                                </ul>
-                            </div>
-                            <button class="project-button">
-                            <span>See this project</span> 
-                                <img src="assets/images/arrow.svg" alt="arrow" class="arrow">
-                            </button>
-                        </div>
-                    </section>
+          <div><img src="assets/desktop-img/backgroung-3.svg" alt="" /></div>
+        </div>
+        <div class="fourth-container">
+          <section class="containing-6">
+            <div class="rectangle_21">
+              <img src="assets/images/rectangle_26.svg" alt="rectangle_21" />
+              <div class="rectangle_22">
+                <span class="project-name">project name goes here</span>
+                <div>
+                  <ul class="project-languages">
+                    <li class="language-css">HTML/CSS</li>
+                    <li class="language-css">Ruby on Rails</li>
+                    <li class="language-css">JavaScript</li>
+                  </ul>
+                </div>
+                <button type="button" class="project-button">
+                  <span>See this project</span>
+                  <img
+                    src="assets/images/arrow.svg"
+                    alt="arrow"
+                    class="arrow"
+                  />
+                </button>
+              </div>
+            </div>
+          </section>
+        </div>
+      </section>
```

Comment:
> - [ ] Please kindly ensure that your design sticks with the Figma design as much as possible. Also kindly ensure that each card container on hover is correctly aligned and also the list of  "project languages" is aligned properly on the mobile screen too. This will make your users a good experience and feel of your website.👍
> 
> Figma design:
> 
> ![4-1](https://github.com/axcell-0/Thierrys-portfolio/assets/40009354/d6d01ff3-9270-4e3d-9332-e5f1d52ef619)
> 
> Your design:
> 
> ![4](https://github.com/axcell-0/Thierrys-portfolio/assets/40009354/d3c6556f-b920-4b61-8a04-d0c2d0be2be7)
>